src/Build/BackEnd/Components/Logging/LoggingContext.cs

```diff
@@ -106,6 +109,8 @@ protected set
             }
         }
 
+        internal bool HasLoggedErrors { get { return _hasLoggedErrors; } set { _hasLoggedErrors = value; } }
```

Comment:
Should we add some more scoped unit tests to `LoggingContext_Tests`? It doesn't look like there are any tests that cover the methods you're changing :(

Reply:
Agreed. Any suggestions for testing this specifically?